opentracing/ext/tags.go

```diff
@@ -9,44 +9,38 @@ var (
 	// the other side/service in a peer-to-peer communications, like an RPC call.
 
 	// PeerService records the service name of the peer
-	PeerService = &stringTag{"peer.service"}
+	PeerService = stringTag("peer.service")
 
 	// PeerHostname records the host name of the peer
-	PeerHostname = &stringTag{"peer.hostname"}
+	PeerHostname = stringTag("peer.hostname")
 
 	// PeerHostIPv4 records IP v4 host address of the peer
-	PeerHostIPv4 = &uint32Tag{"peer.ipv4"}
+	PeerHostIPv4 = uint32Tag("peer.ipv4")
 
 	// PeerHostIPv6 records IP v6 host address of the peer
-	PeerHostIPv6 = &stringTag{"peer.ipv6"}
+	PeerHostIPv6 = stringTag("peer.ipv6")
 
 	// PeerPort records port number of the peer
-	PeerPort = &uint16Tag{"peer.port"}
+	PeerPort = uint16Tag("peer.port")
 )
 
-type stringTag struct {
-	Key string
-}
+type stringTag string
```

Comment:
If others are opposed to writing `string(theTag)` instead of `theTag.Key` I can revert that part of this change; but I def. think we should not be using pointer values for these.